src/biogeochem/CNBalanceCheckMod.F90

```diff
@@ -92,11 +110,17 @@ subroutine BeginCNBalance(this, bounds, num_soilc, filter_soilc, &
     !
     ! !LOCAL VARIABLES:
     integer :: fc,c
+    integer :: g
+    integer :: begg, endg
     !-----------------------------------------------------------------------
 
     associate(                                            & 
-         col_begcb    => this%begcb_col                  , & ! Output: [real(r8) (:)]  (gC/m2) carbon mass, beginning of time step
-         col_begnb    => this%begnb_col                  , & ! Output: [real(r8) (:)]  (gN/m2) nitrogen mass, beginning of time step 
+         grc_begcb    => this%begcb_grc                  , & ! Output: [real(r8) (:)]  (gC/m2) gridcell carbon mass, beginning of time step
+         grc_begnb    => this%begnb_grc                  , & ! Output: [real(r8) (:)]  (gN/m2) gridcell nitrogen mass, beginning of time step
+!        totgrcc      => cnveg_carbonstate_inst%totc_grc , & ! Input:  [real(r8) (:)]  (gC/m2) total gridcell carbon, incl veg and cpool
+!        totgrcn      => cnveg_nitrogenstate_inst%totn_grc, & ! Input:  [real(r8) (:)]  (gN/m2) total gridcell nitrogen, incl veg
```

Comment:
In this first attempt to set up the framework for gridcell-level balance checks for carbon, I have commented out  my own references to `totgrcc` and `totgrcn` throughout, because I may not need them. Instead, I think I just need:
1) all gridcell-level pools and fluxes not accounted for at the column level and
2) the column-level balance error converted to the gridcell level using `call c2g`